package/src/components/AudioPlayer/Context/__tests__/reducer.test.ts

```
@@ -0,0 +1,195 @@
+    expect(next.curIdx).toBe(2);
+  });
+
+  it("picks a different random index when repeatType SHUFFLE", () => {
+    const state = {
+      ...baseState,
+      curIdx: 0,
+      curAudioState: { ...baseState.curAudioState, repeatType: "SHUFFLE" as const },
+    };
+    // Run multiple times to verify randomness never returns same index
+    const results = Array.from({ length: 50 }, () =>
+      audioPlayerReducer(state, { type: "NEXT_AUDIO" }).curIdx
```

ESLint

[error] Replace `{·length:·50·},·()·=>⏎·····` with `⏎······{·length:·50·},⏎······()·=>`

(prettier/prettier)